### DISCUSSION TITLE
Unable to initialize nextflow environment

### DISCUSSION THREAD

**Discussion by tabatakenshiro** [nextflow-io/nextflow#3319] [Q&A]
## Bug report 

I can not gitclone nextflow.

I've done the following command.
chmod 777 nextflow

I have no idea how to solve this error.

### Program output 
user@Ubuntu:~/nextflow$ curl -fsSL https://get.nextflow.io | bash
CAPSULE: Downloading dependency org.multiverse:multiverse-core:jar:0.7.0
CAPSULE: Downloading dependency org.slf4j:jul-to-slf4j:jar:1.7.36
CAPSULE: Downloading dependency org.slf4j:jcl-over-slf4j:jar:1.7.36
CAPSULE: Downloading dependency commons-io:commons-io:jar:2.11.0
CAPSULE: Downloading dependency org.codehaus.jsr166-mirror:jsr166y:jar:1.7.0
CAPSULE: Downloading dependency commons-codec:commons-codec:jar:1.15
CAPSULE: Downloading dependency com.google.guava:listenablefuture:jar:9999.0-empty-to-avoid-conflict-with-guava
CAPSULE: Downloading dependency com.beust:jcommander:jar:1.35
CAPSULE: Downloading dependency io.nextflow:nf-httpfs:jar:22.10.0
CAPSULE: Downloading dependency org.codehaus.groovy:groovy:jar:3.0.13
CAPSULE: Downloading dependency io.nextflow:nextflow:jar:22.10.0
CAPSULE: Downloading dependency ch.grengine:grengine:jar:1.3.0
CAPSULE: Downloading dependency jline:jline:jar:2.9
CAPSULE: Downloading dependency ch.qos.logback:logback-core:jar:1.2.11
CAPSULE: Downloading dependency com.github.zafarkhaja:java-semver:jar:0.9.0
CAPSULE: Downloading dependency org.jsoup:jsoup:jar:1.14.3
CAPSULE: Downloading dependency org.yaml:snakeyaml:jar:1.30
CAPSULE: Downloading dependency org.codehaus.groovy:groovy-nio:jar:3.0.13
CAPSULE: Downloading dependency javax.mail:mail:jar:1.4.7
CAPSULE: Downloading dependency org.eclipse.jgit:org.eclipse.jgit:jar:6.2.0.202206071550-r
CAPSULE: Downloading dependency com.google.code.findbugs:jsr305:jar:3.0.2
CAPSULE: Downloading dependency org.codehaus.groovy:groovy-templates:jar:3.0.13
CAPSULE: Downloading dependency io.nextflow:nf-commons:jar:22.10.0
CAPSULE: Downloading dependency commons-lang:commons-lang:jar:2.6
CAPSULE: Downloading dependency org.iq80.leveldb:leveldb-api:jar:0.12
CAPSULE: Downloading dependency org.codehaus.gpars:gpars:jar:1.2.1
CAPSULE: Downloading dependency ch.qos.logback:logback-classic:jar:1.2.11
CAPSULE: Downloading dependency org.slf4j:log4j-over-slf4j:jar:1.7.36
CAPSULE: Downloading dependency com.google.guava:guava:jar:31.1-jre
CAPSULE: Downloading dependency org.objenesis:objenesis:jar:2.1
CAPSULE: Downloading dependency com.esotericsoftware.kryo:kryo:jar:2.24.0
CAPSULE: Downloading dependency org.slf4j:slf4j-api:jar:1.7.36
CAPSULE: Downloading dependency org.codehaus.groovy:groovy-xml:jar:3.0.13
CAPSULE: Downloading dependency com.google.errorprone:error_prone_annotations:jar:2.11.0
CAPSULE: Downloading dependency org.apache.ivy:ivy:jar:2.3.0
CAPSULE: Downloading dependency dev.failsafe:failsafe:jar:3.1.0
CAPSULE: Downloading dependency com.google.j2objc:j2objc-annotations:jar:1.3
CAPSULE: Downloading dependency javax.activation:activation:jar:1.1.1
CAPSULE: Downloading dependency org.pf4j:pf4j:jar:3.4.1
CAPSULE: Downloading dependency org.iq80.leveldb:leveldb:jar:0.12
CAPSULE: Downloading dependency org.codehaus.groovy:groovy-json:jar:3.0.13
CAPSULE: Downloading dependency com.google.guava:failureaccess:jar:1.0.1
CAPSULE: Downloading dependency com.googlecode.javaewah:JavaEWAH:jar:1.1.13
CAPSULE: Downloading dependency com.google.code.gson:gson:jar:2.2.4
CAPSULE: Downloading dependency org.checkerframework:checker-qual:jar:3.12.0
CAPSULE: Downloading dependency org.pf4j:pf4j-update:jar:2.3.0
CAPSULE EXCEPTION: Error resolving dependencies. while processing attribute Allow-Snapshots: false (for stack trace, run with -Dcapsule.log=verbose)
Unable to initialize nextflow environment

### Environment 

* Java version:  "11.0.16" 2022-07-19
* Operating system: [Linux, Ubuntu 20.04.4 LTS]
* Bash version: GNU bash, version 5.0.17(1)-release (x86_64-pc-linux-gnu)

**Comment by ernstki** [comment]
I had a similar problem, but not this _exact_ problem, so unfortunately I don't have a solution for you.

However, one thing you can do to get more details is download the installation script to a local file

```bash
curl -fsSL https://get.nextflow.io > get-nextflow

# make it executable
chmod a+x get-nextflow
```

then run it with `-Dcapsule.log=verbose` as suggested in the error output

```
./get-nextflow -Dcapsule.log=verbose
```

Perhaps the extra debug output from Capsule will cue you in to the exact problem.
